apisix/runner/plugin/core.py

```diff
@@ -14,18 +14,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import os
 import importlib
 from pkgutil import iter_modules
 
 
-def instances() -> dict:
-    modules = iter_modules(__import__("plugins").__path__)
+def filter(configs: dict, request, response) -> None:
+    for name in configs:
+        plugin = configs.get(name)
+        if not plugin:
+            print("plugin undefined.")
+            continue
+        try:
+            plugin.filter(request, response)
+        except Exception as e:
+            print(e)
```

Comment:
The error message should be more detailed

Reply:
fixed